### DISCUSSION TITLE
How to change root directory with lua script using api

### DISCUSSION THREAD

**Discussion by fcacicAi** [nvim-tree/nvim-tree.lua#2649] [General]
openNewRoot = function (root)
    local api = require("nvim-tree.api")

    api.tree.open({
      path=root
      current_window=true
    })

I would like to have function that can change the root of current nvim-tree view.

So i can easily shift between projects. 
But this does not change the view in nvim-tree.

**Comment by gegoune** [comment]
Is `:h sync_root_with_cwd` what you re after?
